src/modules/admin/admin.controller.ts

```diff
@@ -63,4 +69,13 @@ export class AdminController {
       limit,
     );
   }
+
+  @Get('products')
```

Comment:
@ApiOperation

Reply:
Added description and summary for product route. How it looks on swagger you can see on the screenshot above